src/NuGet.Core/NuGet.Commands/MSBuildProjectFactory.cs

```diff
@@ -57,7 +57,24 @@ public ILogger Logger
 
         public IMachineWideSettings MachineWideSettings { get; set; }
 
+        /// <summary>
+        /// Creates a MSBuild project factory with data from NuGet <see cref="Commands.PackArgs"/> object
+        /// </summary>
+        /// <param name="packArgs">NuGet arguments for creating a package</param>
+        /// <param name="path">Not used</param>
+        /// <returns>A <see cref="MSBuildProjectFactory"/> object</returns>
+        /// <remarks>This methods is used in <see cref="PackCommandRunner.CreateProjectFactory"/> delegate</remarks>
```

Comment:
Assuming no references to this method in the product code, can we mark this method obsolete?

Reply:
No. This is a delegate method implemetation of https://github.com/NuGet/NuGet.Client/blob/563216c35ce91f8c2bdb5ca1d102e639107a40b5/src/NuGet.Core/NuGet.Commands/CommandRunners/PackCommandRunner.cs#L24

Reply:
Removed, since other builds are unnafected.